config/config.js

```diff
@@ -0,0 +1,30 @@
+import env from "../config/env.generated.json";
+
+let config = {
+  userLoggedInStatus: false,
+  username: ``
+};
+
+export default {
+  logInUrl: {
+    get: function(redirectUrl = env.ORIGIN) {
+      return `${env.PULSE_API}/login?original_url=${encodeURIComponent(redirectUrl)}`;
```

Comment:
There's server side generation going on as well so we can't simply set it to `window.location.toString()`